grease-cli/src/Grease/Cli.hs

```diff
@@ -57,12 +57,14 @@ boundedEnumMetavar _ = Opt.metavar $ varShowS ""
 
 entrypointParser :: Opt.Parser Entrypoint
 entrypointParser =
-  addressParser
-    Opt.<|> symbolParser
-    Opt.<|> coreDumpParser
-    Opt.<|> addressStartupOvParser
-    Opt.<|> symbolStartupOvParser
+  inGroup addressParser
+    Opt.<|> inGroup symbolParser
+    Opt.<|> inGroup coreDumpParser
+    Opt.<|> inGroup addressStartupOvParser
+    Opt.<|> inGroup symbolStartupOvParser
```

Comment:
Possibly silly `optparse-applicative` question: would it be equivalent to write `Opt.parserOptionGroup "Entrypoint options" (addressParser <|> symbolParser <|> ...)` instead? This would be shorter and avoid duplication.